examples/kubernetes/static_provisioning/README.md

```diff
@@ -7,6 +7,7 @@ This example shows how to make a static provisioned Mountpoint for S3 persistent
 - `s3_express_specify_az.yaml` - same as above, but this uses a [S3 Express One Zone](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) directory bucket and shows how to specify the availability zone (AZ) of the [persistent volume](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) to co-locate the pods with the bucket for lower latency access
 - `multiple_buckets_one_pod.yaml` - same as above, with multiple buckets mounted in one pod. Note: when mounting multiple buckets in the same pod, the `volumeHandle` must be unique as specified in the [CSI documentation](https://kubernetes.io/docs/concepts/storage/volumes/#csi).
 - `multiple_pods_one_pv.yaml` - same as above, with multiple pods mounting the same persistent volume, in Deployment kind, that can be used to scale pods, using HPA or other method to create more replicas
+- `caching.yaml` - shows how to configure mountpoint to use a cache directory. See the [Mountpoint documentation](https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#caching-configuration) for more details on caching options. Please thumbs up [#11](https://github.com/awslabs/mountpoint-s3-csi-driver/issues/141) or add deatils about your use case if you want improvements in this area.
```

Comment:
I would add something about the process of creating the cache in the root host similar to the comment inside of the yaml `# specify cache directory, relative to root host filesystem`. I think this [issue](https://github.com/awslabs/mountpoint-s3-csi-driver/issues/119) had some info on it as well.

Reply:
I'm planning to overhaul the structure of these docs soon to add more detail, but with the current format the readme just has a quick overview and the detail's in the yaml. So my preference would be to merge this as is and follow up with a more comprehensive update, but don't have a strong opinion one way or the other.